insolar/flow/_example/basic.go

```diff
@@ -0,0 +1,169 @@
+//
+// Copyright 2019 Insolar Technologies GmbH
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+package _example
+
+import (
+	"bytes"
+	"context"
+	"fmt"
+
+	"github.com/ThreeDotsLabs/watermill/message"
```

Comment:
do we still need example in repo?
If this code is necessary - make dep ensure for this dependency